lib/android_build/maesdk/src/main/java/com/microsoft/applications/events/ILogManager.java

```diff
@@ -68,4 +68,10 @@
   public boolean registerPrivacyGuard();
 
   public boolean unregisterPrivacyGuard();
+  
+  public void pauseActivity();
+  public void resumeActivity();
+  public void waitPause();
+  public boolean startActivity();
```

Comment:
Very minor nit, can this be on top? Then the list feels order in that all the voids are together and the first method is `startActivity` and last is `endActivity`

Reply:
Below as well.